browser/search_engines/search_engine_tracker.cc

```diff
@@ -27,14 +27,16 @@ void RecordSearchEngineP3A(const GURL& search_engine_url,
     answer = SearchEngineP3A::kBing;
   } else if (type == SEARCH_ENGINE_QWANT) {
     answer = SearchEngineP3A::kQwant;
+  } else if (type == SEARCH_ENGINE_YAHOO) {
+    answer = SearchEngineP3A::kYahoo;
   } else if (type == SEARCH_ENGINE_OTHER) {
     if (base::EndsWith(search_engine_url.host(), "startpage.com",
                        base::CompareCase::INSENSITIVE_ASCII)) {
       answer = SearchEngineP3A::kStartpage;
     }
   }
 
-  UMA_HISTOGRAM_ENUMERATION("Brave.Search.DefaultEngine", answer);
+  UMA_HISTOGRAM_ENUMERATION("Brave.Search.DefaultEngine.2", answer);
```

Comment:
we should actually use `kDefaultSearchEngineMetric` here, apparently this was missing from the very beginning :D

Reply:
also mind updating?

Reply:
Updated! Thanks 😄